calico-enterprise/multicluster/federation/kubeconfig.mdx

```diff
@@ -216,46 +227,139 @@ We'll now create the RemoteClusterConfigurations that establish synchronization
+
+   You can achieve this by creating a disabled IP pool in cluster A for each CIDR in cluster B. This IP pool should have NAT-outgoing disabled. For example:
+
+    ```yaml
+    apiVersion: projectcalico.org/v3
+    kind: IPPool
+    metadata:
+      name: clusterB-main-pool
+    spec:
+      cidr: <Cluster B CIDR>
+      disabled: true
+    ```
+
+1. [Validate](#check-remote-cluster-connection) the that the remote cluster connection can be established.
+
+1. Repeat the above steps, switching cluster A and cluster B.
+
+
+After completing the above steps for all cluster pairs in the cluster mesh, your clusters should now be ready to utilize remote-identity-aware policy and federated services.
+
+</TabItem>
+</Tabs>
+
+:::caution
  This tutorial sets up RemoteClusterConfigurations in both directions. This is required for {{prodname}} to manage multi-cluster networking, and also ensures you can write identity-aware policy on both sides of a cross-cluster connection. Unidirectional connections can be made at your own discretion.
 :::
 
-### Validate federation and multi-cluster networking
-#### RemoteClusterConfiguration & federated endpoint identity
-[calicoq](../../operations/clis/calicoq/installing) can be used to ensure that a local cluster has connected to it's configured remote clusters. Use the following command:
+### Switch to multi-cluster networking
+The steps above assume that you are configuring federated endpoint identity and multi-cluster networking for the first time. If you already have federated endpoint identity and are looking to utilize multi-cluster networking, the steps below outline the changes that must be made if you had previously set up federated endpoint identity prior to the introduction of multi-cluster networking:
+
+1. Validate that all [requirements](#calico-enterprise-multi-cluster-networking) for multi-cluster networking have been met.
+2. Update the ClusterRole in each cluster in the cluster mesh using the RBAC manifest found in [Create kubeconfig files](#create-kubeconfig-files)
+3. Update all RemoteClusterConfigurations to set `Spec.OverlayRoutingMode` to `Enabled`
+4. Ensure that all RemoteClusterConfigurations are bidirectional, i.e. ensure that RemoteClusterConfigurations were established following the [instructions](#create-remoteclusterconfigurations) for in both directions for each cluster pair.
+5. If you had previously created disabled IP pools to prevent NAT outgoing from applying to remote cluster destinations, those disabled IP pools are no longer needed when using multi-cluster networking and must be deleted.
+
+
+### Validate federated endpoint identity & multi-cluster networking
+#### Validate RemoteClusterConfiguration & federated endpoint identity
+##### Check remote cluster connection
+You can validate in a local cluster that Typha has synced to the remote cluster through the [Prometheus metrics for Typha](../../reference/component-resources/typha/prometheus#metric-reference).
```

Comment:
```suggestion
To verify that Typha has synced a local cluster to the remote cluster, you can: 

- View [Prometheus metrics for Typha](../../reference/component-resources/typha/prometheus#metric-reference)
- Check Typha logs using this command: `kubectl logs deployment/calico-typha -n calico-system | grep "Sending in-sync update"`

Reply:
I'm not sure the log check can be condensed onto one line. It needs the context of "You should see an entry for each RemoteClusterConfiguration in the local cluster", which feels like too much for one line.

Leaving as-is for now, we can revisit on next pass